### ISSUE TITLE
E2EE: default logging configuration logs decrypted messages by default

### ISSUE THREAD

**Issue by lbeltrame** [mujx/nheko#339]
### System:

- Nheko version: 0.43-8704265
- Installation method: Source
- Operating System: openSUSE Tumbleweed
- Qt version: 5.11
- C++ compiler: GCC 8
- Desktop Environment:KDE Plasma 5.13.80 (git master version)

### Actual behavior

The default logging categories seem to output at least to stdout the contents of decrypted messages, as shown (personal bits removed):

```
[2018-06-18 14:09:53.004] [crypto] [info] decrypted data: 
 {
  "content": {
    "body": "test",
    "msgtype": "m.text"
  },
  "event_id": "EVENT_ID",
  "origin_server_ts": TIMESTAMP,
  "room_id": "ROOM_ID",
  "sender": "@my_user_name:example.com",
  "type": "m.room.message",
  "unsigned": {
    "age": 223
  }
}
```

### Expected behavior

Outputting decrypted messages should be on the Debug logging level, rather than Info

### Steps to reproduce

1. Launch nheko in a terminal
2. Type or receive an event in an encrypted room
3. Watch nheko's standard output

**Comment by mujx** [comment]
This has been fixed by 4514d13948158ee8f6a7cb22bedd5b462802c2d1 which only logs if debug is the specified logging level (currently only possible to be changed manually)

**Comment by lbeltrame** [comment]
Can't reproduce anymore with latest master. Closing.
